dashboard/app/models/script.rb

```diff
@@ -521,6 +525,14 @@ def hoc_finish_url
     end
   end
 
+  def csf_finish_url
+    if name == Script::TWENTY_HOUR_NAME
+      CDO.code_org_url "/congrats/accelerated"
+    else
+      CDO.code_org_url "/congrats/#{name}"
+    end
+  end
```

Comment:
We should probably define Script::ACCELERATED_NAME = "accelerated" and use that here. Also, continuing to preach the gospel of ruby style, how about structuring it like this:
```
name = Script::ACCELERATED_NAME if name == Script::TWENTY_HOUR_NAME
CDO.code_org_url "/congrats/#{name}"
```
disclaimer: I'm not sure I can give the best explanation for why this style is preferred, but it's how reviewers of my ruby code have recommended that I style things. It seems a tiny bit less repetitive perhaps?